src/cavendish_particle_tracks/_widget.py

```diff
@@ -42,16 +43,18 @@ def __init__(self, napari_viewer: napari.viewer.Viewer):
         self.viewer = napari_viewer
 
         # define QtWidgets
+        # why is this self. and the others added after?
```

Comment:
This is related with how the tests are implemented.  For combobox, you need to be able to set the index.... so this is the way I went around it.